client/src/components/Nav/index.js

```diff
@@ -1,90 +1,90 @@
-import React, { Component } from 'react'
-import { Link } from 'react-router-dom'
-import logo from '../img/piggybusiness_logo_01.png';
-import '../../App.css';
-import axios from 'axios';
+import React, { Component } from "react";
+import logo from "../img/piggybusiness_logo_01.png";
```

Comment:
what type of logo is this? Project logo? Company Logo? logo is very vague. it can be any type of logo. is it the small logo? maybe you guys have 5 different logos and they are all the same logo but different sizes. i dont know what logo just means.